source/configuring_domain/02_managing_legal_information/legals.rst

```diff
@@ -64,9 +64,26 @@ To specify the Terms and Conditions and the Privacy Policy of a portal, go to Co
    Fill-in Terms & Conditions and Privacy Policy templates are available, based on the French law (in French and English languages) and the US law (in English language) only. Click the Use a template button under the textbox to use a template. The generated conditions of use can be edited in the textbox afterward.
 
 
+Licenses
+--------
+
+When configuring the metadata of a dataset in the Information tab, the license attached to that dataset must be defined, using a drop-down selection of predefined licenses. To define the licenses listed in the drop-down selection, go to Configuration > Legals. In this subsection of the back office menu, look for the "Licenses" area.
```

Comment:
> the license attached to that dataset must be defined

The license is not mandatory, the default choice is "no license". It's always a really good idea to configure it though, so I'm not sure if the best term is "should" or "can".